model_zoo/lbann2.cpp

```diff
@@ -77,7 +77,7 @@ int main(int argc, char *argv[]) {
 
     // When using checkpoint states, skip training as those could be the result
     // of checkpointing by steps.
-    if (!opts->has_string("ckpt_dir")){
+    if (!opts->has_string("no_model1_train")){
```

Comment:
@JaeseungYeom I believe you had set this in a previous commit to, by default, not train model1 when loading from ckpt. Would it work for you to have this as a command line option `--no_model1_train` instead of the default behavior?

Reply:
That would be a better option.